bl31/runtime_svc.c

```diff
@@ -80,33 +84,32 @@ static int32_t validate_rt_svc_desc(rt_svc_desc_t *desc)
  ******************************************************************************/
 void runtime_svc_init(void)
 {
-	int32_t rc = 0;
-	uint32_t index, start_idx, end_idx;
-	uint64_t rt_svc_descs_num;
+	int rc = 0, index, start_idx, end_idx;
```

Comment:
This change was primarily targeted towards variables that need to be of different word size depending on the Architecture(AArch32/AArch64) it is compiled for. Typically variables holding address (uintptr_t), address range (size_t) were addressed as part of the type rework.

The rest of generic variables were not changed. The `indices` here come under that category.